src/Symfony/Component/HttpFoundation/StreamedResponse.php

```diff
@@ -58,6 +58,10 @@ public function setCallback(callable $callback): static
 
     public function getCallback(): \Closure
     {
+        if (null === $this->callback) {
+            throw new \LogicException('The Response callback must not be null.');
```

Comment:
Not sure, but the LogicException should be used from the Component and not from the root namespace